ansible-playbook/roles/gophish/files/config.json

```diff
@@ -0,0 +1,17 @@
+{
+    "admin_server" : {
+        "listen_url" : "127.0.0.1:3333",
+        "use_tls" : false,
```

Comment:
Could we set this to `true` to match the existing config file? https://github.com/gophish/gophish/blob/master/config.json